Pipfile

```diff
@@ -6,8 +6,9 @@ name = "pypi"
 [packages]
 flask = "*"
 gunicorn = "*"
-SPARQLWrapper = "*"
 jsonschema = "*"
+sparqlwrapper = "*"
+pytest = "*"
```

Comment:
we don't need this here. We probabaly need to update the readme but the way it works is:

- `pipenv install` - installs the libraries in packages
- `pipenv install -d` (d for dev) installs the libraries in "packages" **and** the libraries in "dev-packages".

You can just use the latter and remove this duplicate pytest entry from "packages".

It's a separation principle, we want "packages" to only contain libraries used when running in production, keeps things nice and clean.

Could you likewise move jsonschema down to dev-packages as well (and confirm the tests run ok please).